src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/payments.html.twig

```diff
@@ -38,6 +38,22 @@
           <strong>{{ orderForViewing.payments.paidAmount }}</strong>
           {{ 'paid instead of'|trans({}, 'Admin.Orderscustomers.Notification') }}
           <strong>{{ orderForViewing.payments.amountToPay }}</strong>
+
+          {% if orderForViewing.brothers.brothers is not empty %}
+
+            {% set firstBrother = orderForViewing.brothers.brothers|first %}
+            {% if orderForViewing.brothers.brothers|length == 1 %}
+              <br/>{{ 'This warning also concerns order '|trans({}, 'Admin.Orderscustomers.Notification') }}
```

Comment:
Is the extra space at the end added on purpose so that the order ID is not glued to the last word?

Reply:
Yes. There will be a list of IDs after this sentence, like this: "This warning also concerns order #3398282 #7278282 #93993"
<img src="https://user-images.githubusercontent.com/3830050/84295907-845ff700-ab4b-11ea-9e02-e939de0fc0e6.png">

Reply:
I thought this notification was just for a single order ID since there is another wording right after: _This warning also concerns the following orders:_. What is the difference between the two?

Regarding the extra space at the end, it is not a very good practice because the string will be translated afterward and translators might not notice it or just forget to keep it. And therefore it might not be displayed properly in the back office. I suggest adding the `:` punctuation sign instead: 1/ it will be consistent with the other notification, 2/ it will ensure a correct display. So _This warning also concerns order:_ seems better.

Reply:
Done ! in https://github.com/PrestaShop/PrestaShop/pull/19704/commits/8a996d84c86bdb2923f42daf0124f04d402f5d80